src/Interpreters/InterpreterFactory.cpp

```diff
@@ -233,7 +233,7 @@ InterpreterFactory::InterpreterPtr InterpreterFactory::get(ASTPtr & query, Conte
     else if (query->as<ASTExplainQuery>())
     {
         const auto kind = query->as<ASTExplainQuery>()->getKind();
-        if (kind == ASTExplainQuery::ParsedAST || kind == ASTExplainQuery::AnalyzedSyntax)
+        if (kind == ASTExplainQuery::ParsedAST)
             context->setSetting("allow_experimental_analyzer", false);
```

Comment:
Why do we need to disable the analyzer to show parsed AST?

Reply:
I haven't tried to enable it (it's a different type of explain), probably it should be addressed separately